README.md

```diff
@@ -140,7 +141,9 @@ Here's an example of how you might document a 4x function:
      end
 ```
 
-And here's an example of how you might document a class:
+### Classes / Defined Types
```

Comment:
I assume defined types work exactly the same (I thought I proved that out already).